desktop/settings/linux.md

```diff
@@ -126,11 +128,11 @@ Type a JSON configuration file in the box to configure the daemon settings. For
 
 Click **Apply & Restart** to save your settings and restart Docker Desktop.
 
-## Experimental Features
+## Beta Features
```

Comment:
This will not be released in 4.12 though 😬

Reply:
Ah, should've spotted that.... I haven't had my morning coffee yet ☕ ....will amend now and reopen your PR